src/main/java/com/team701/buddymatcher/controllers/users/UserController.java

```diff
@@ -136,7 +136,7 @@ public UserController(UserService userService, ModelMapper modelMapper) {
         }
     }
 
-    @Operation(summary = "Post method insert a user as a buddy")
+    @Operation(summary = "Delete method to delete a user's buddy")
```

Comment:
This method operation summary was incorrect, so was changed

Reply:
ty